### PR TITLE
Added HomePage, Navbar , Header 

### LINKED ISSUE #2: Home and Our Team pages
### Description
This GitHub issue is created to track the implementation of "Home" and "Our Team" pages in the project.

### Tasks
  - [ ] Create proper components for the Pages like header, footer, page wrapper etc
  - [ ] Design the page and components so that the page looks good, we prefer you to use figma for this.
  - [ ] make the pages responsive for both pc and mobile.

### Additional Information
- The pages should be styled using Tailwind CSS.
- Ensure proper navigation links are added for easy access to these pages.
- Feel free to create your onw assets in figma or from other sources

### Deadline
[TBD]

**Assigned Developers:**
- [ ] TBD

### PR DESCRIPTION
**Pull request by A-nirvana**
I have added the Homepage the navbar and the header  just like the provided design. I have also made it responsive to work on phone.
Screenshot (30)

### CODE REVIEW

**Comment by A-nirvana** [comment]
@itsKhadeer I am building the events page. Tell me what I can improve in this page and give me ideas for the next one please.

**Comment by itsKhadeer** [comment]
image

-  [ ] fix this overflow in the teams page and also reduce the size of the blue box,

- [ ] in the swiper, don't fix the blue box as background img, make it swipe along with the content, like a card.

-  [ ] also the teams page ui is not responsive for iphone se, take iphone se as reference for making the ui mobile responsive, fix it.

- [ ] Add teams page in navbar and also remove unnecessary fields in navbar and add the important onces,

- [ ] Add animation for the navbar (when it opens and closes) 

- [ ] try to add more components instead of writing your whole code in the pages, for example you can make the swiper as a component.

image
- [ ] fix this over lapping of buttons in the home page

image
- [ ] fix this overflow here in the above image

- [ ] also add a line below the "TEAMS" heading in the our teams page, like in the home page.

- [ ] after making your changes please test properly.

File: src/pages/OurTeam/OurTeam.tsx
Comment (itsKhadeer) [inline src/pages/OurTeam/OurTeam.tsx:1]: try to make the swiper as a seperate component instead of directly implementing it here

**Review by itsKhadeer — CHANGES_REQUESTED** [review]
please complete the changes

**Comment by A-nirvana** [comment]
@itsKhadeer I have made the requested changes please review these changes and tell me if there's any problem.

**Review by itsKhadeer — APPROVED** [review]

**Pull request merged**